config/default.yml

```
@@ -3284,6 +3284,11 @@ Style/NilComparison:
     - predicate
     - comparison
 
+Style/NonLocalControlFlowInTernary:
+  Description: 'checks for non-local control flow in ternary expressions.'
+  Enabled: true
+  VersionAdded: '0.76'
```

It looks like the version needs to be updated.

```suggestion
  VersionAdded: '0.81'
```